portfolio/src/main/webapp/style.css

```diff
@@ -9,8 +9,13 @@
     margin-top: 1em;
 }
 
-#project-pic{
-    max-width: 45%
+.project-pic {
+    height: 400px;
+    margin-top: 15%;
```

Comment:
Just as a warning, this (alongside the 1/3 column thing) is causing the pictures to get warped when the browser's width is shrunk (it's actually warped even at fullsize on my big 24'' monitor!)

Reply:
Specifically try the tweeter or MMR images.

Reply:
i think this because i have pictures with different size and i want them to have the same height so that the page won't jump when switching between images.